src/index.js

```diff
@@ -20,17 +20,21 @@ const pluginTranslations = [
 	},
 ];
 
-registerGutenberg( {
-	beforeInitCallback: () => {
-		// We have to lazy import the setup code to prevent executing any code located
-		// at global scope before the editor is initialized, like translations retrieval.
-		require( './setup' ).default();
+export default function registerGutenbergMobile() {
```

Comment:
I had to expose the register function mainly for testing purposes, so we can initialize Gutenberg Mobile on each test without having to reimport the module.